pycbc/workflow/coincidence.py

```diff
@@ -546,6 +546,68 @@ def setup_interval_coinc(workflow, hdfbank, trig_files, stat_files,
     logging.info('...leaving coincidence ')
     return statmap_files
 
+def setup_multiifo_interval_coinc_inj(workflow, hdfbank, full_data_trig_files, inj_trig_files,
+                                      stat_files, veto_file, veto_name,
+                                      out_dir, pivot_ifo, fixed_ifo, tags=None):
+    """
+    This function sets up exact match multiifo coincidence for injections
+    """
+    if tags is None:
+        tags = []
+    make_analysis_dir(out_dir)
+    logging.info('Setting up coincidence')
+
+    if len(hdfbank) != 1:
+        raise ValueError('Must use exactly 1 bank file for this coincidence '
+                         'method, I got %i !' % len(hdfbank))
+    hdfbank = hdfbank[0]
+    # Wall time knob and memory knob
+    factor = int(workflow.cp.get_opt_tags('workflow-coincidence', 'parallelization-factor', tags))
+
+    ffiles = {}
+    ifiles = {}
+    ifos, files = full_data_trig_files.categorize_by_attr('ifo')
+    for ifo, file in zip(ifos, files):
```

Comment:
same remarks as for previous iterator over `ifo, file` : don't use 'file' as a variable name. 
Also I think you can do, as was done in the workflow code, `zip(*full_data_trig_files.categorize_by_attr('ifo'))` 
& save a few lines of code

Reply:
ok, ian pointed out what was the problem here, I've added a commit directly which should correctly compress these into a single assignment